tsconfig.json

```diff
@@ -18,6 +18,7 @@
     "./config/config.js",
     "./models/tagpost.js",
     "./models/savedPosts.js",
+    "./models/pinnedPosts.js",
```

Comment:
This will have the same bug as the saved posts — the file is `pinnedposts.js` #csharpisms 😊